src/grocker/cleanners.py

```diff
@@ -0,0 +1,102 @@
+# -*- coding: utf-8 -*-
+# Copyright (c) Polyconseil SAS. All rights reserved.
+
+from __future__ import absolute_import, division, print_function, unicode_literals
+
+import itertools
+import logging
+
+import docker.errors
+import packaging.version
+
+from . import __version__
+
+logger = logging.getLogger(__name__)
+
+
+def version_compare(obj, version):
```

Comment:
This function name is quite unclear.
You could create a function `get_grocker_version` on one end and build a `packaging.specifiers.SpecifierSet` from `__version__`